HISTORY.rst

```diff
@@ -14,6 +14,22 @@ History
     - ``.forgiven``
 - Added ``Invoice.status`` and ``enums.InvoiceStatus`` (#1020).
 - Fixed str(Account) crash when settings or business_profile were NULL (#1104).
+- Dropped previously-deprecated ``enums.InvoiceStatus`` (#1020).
+- Added new ``Invoice`` fields (#1020, #1087):
+    - ``.discount``
+    - ``.default_source``
+    - ``.status``
+- Added new ``Subscription`` fields (#1087):
+    - ``.default_payment_method``
+    - ``.default_source``
+    - ``.next_pending_invoice_item_invoice``
+    - ``.pending_invoice_item_interval``
+    - ``.pending_update``
+    - ``.start_date``
+- Deprecated the following fields - will be removed in 2.4 (#1087):
```

Comment:
Minor tweak - can you move deprecations/dropped fields this to the top of this version's changes? (maybe just after the drop of python/django version), so users are more likely to notice.